exporter/jaeger/jaegerthrifthttpexporter/factory_test.go

```diff
@@ -98,7 +98,7 @@ func TestFactory_CreateTraceExporterFails(t *testing.T) {
 					NameVal: typeStr,
 				},
 			},
-			errorMessage: "\"jaeger_thrift_http\" config requires a valid \"url\": parse : empty url",
+			errorMessage: "\"jaeger_thrift_http\" config requires a valid \"url\": parse \"\": empty url",
```

Comment:
nit: you may switch to backticks to make this more readable, i.e.
```
errorMessage: `"jaeger_thrift_http" config requires a valid "url": parse "": empty url`,
```

Reply:
I'd prefer to minimize diffs for this pull, but might address it later (in particular to make it tolerant of both go1.13 and go1.14